app/controllers/api/v0/results_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Api::V0::ResultsController < Api::V0::ApiController
+  def personal_records
+    user = User.find_by_id(params.require(:user_id))
```

Comment:
Maybe `find` instead of `find_by_id` so that it throws exception if user ID is not available?

Reply:
(Sorry for not pointing this in my initial review, I was not aware of this knowledge that time)

Reply:
No worries! I have no idea why I was using `find_by_id` there :P